lib/registries/addon/components/registries-navbar/template.hbs

```diff
@@ -1,6 +1,7 @@
 <Navbar
     data-analytics-scope='Navbar'
     @dark={{true}}
+    {{with-branding this.brand.currentBrand}}
```

Comment:
a thing that's missing here is that the (non-osf) branded navbar logo should replace tha entire dropdown (with Registries, Preprints, etc.), not just the OSF logo in the corner -- the entire contents of `<nav.section data-test-left>`

Reply:
Wait so if a current brand is present, then the branded navbar logo (represented as the `--navbar-logo-img-url` css var) will replace the entire dropdown? Just clarifying that the presence of a brand will remove the dropdown (& links in it).

Reply:
more specifically, the presence of a *non-osf* brand (i think/hope it's the only special case for osf, which otherwise should be treated like any other brand) should do exactly as you described, yes.

Reply:
Wait I'm confused. I thought that the navbar-logo-img was the square avatar image?

Reply:
![Screen Shot 2020-03-26 at 3 36 57 PM](https://user-images.githubusercontent.com/118005/77703288-af3ca500-6f77-11ea-8d32-1fa20ffe3d64.png)

Is this right? Shouldn't there be some brand text next to the logo image?

Reply:
oh sorry for the confusion -- the text in the mockup is part of the logo:
<img width="603" alt="Screen Shot 2020-03-27 at 09 10 06" src="https://user-images.githubusercontent.com/6776190/77759440-1ce5e100-700b-11ea-8bd3-89a216c26ad2.png">
^ the dotty circle and `egap` are in the same image -- the logo should be a wider rectangle

Reply:
(though it it is ambiguous and a square logo with branded text might be nice too -- maybe something to check in the design consult today)